contentcuration/contentcuration/views/base.py

```diff
@@ -60,7 +62,11 @@ def base(request):
 
 
 def health(request):
-    return HttpResponse("Healthy!")
```

Comment:
What's the purpose of this change?

Reply:
How odd- I don't remember changing this as part of the file uploader... The db query should be the correct one though as healthz is a db check